<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMC_GUI</class>
 <widget class="QMainWindow" name="KMC_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>623</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>KMC_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_generate">
     <attribute name="title">
      <string>Generate</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>475</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="12" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_A">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Left Contact</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_z_size">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Path to Data</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Frequency (Base, power of 10)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_y_size">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinBox_time_end">
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QPushButton" name="pushButton_path_to_data">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Right Contact</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QSpinBox" name="spinBox_frequency_power">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinBox_thermalization_time">
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Delta Energy Base</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="spinBox_right_contact">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Root Directory</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QCheckBox" name="checkBox_right_contact">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="4">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>delta energy = A*sin(time*perion)+delta energy base</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Thermalization Time</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QCheckBox" name="checkBox_left_contact">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_frequency_base">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Time End</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinBox_left_contact">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="20" column="0" colspan="4">
        <widget class="QPushButton" name="pushButton_generate">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_type">
         <item>
          <property name="text">
           <string>Random</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sphere</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Plane</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_x_size">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_delta_energi_base">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99999.990000000005239</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLabel" name="label_path_to_data">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="spinBox_period"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_launch">
     <attribute name="title">
      <string>Launch</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_models">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>400</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Select models</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_launch">
         <property name="text">
          <string>Launch</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_threads">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Threads</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_add_model">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear_all">
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_analysis">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>711</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_add_root">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Plot heatmap for layer in time</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_layer"/>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_plot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_roots">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QPushButton" name="pushButton_heatmap">
         <property name="text">
          <string>Heatmap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
